src/components/DateRangePickerDay.js

```diff
@@ -98,11 +100,15 @@ export default class Day extends Component<DayProps> {
 
   render() {
     const { state, marking, date } = this.props;
+    const label = format(date.dateString, "dddd, MMMM D");
+    const traits = marking.selected ? ["button", "selected"] : ["button"];
 
     return (
       <TouchableWithoutFeedback
         onPress={this.onPress}
         onLongPress={this.onLongPress}
+        accessibilityTraits={traits}
+        accessibilityLabel={label}
```

Comment:
You also want to use `accessibilityComponentType="button"` for Android.

Reply:
Done in c9bc188